topics/metagenomics/tutorials/metagenomics-binning/turorial.md

```diff
@@ -0,0 +1,418 @@
+
+![Image shows the process of individual assembly on two strains and five samples, after individual assembly of samples two samples are chosen for de-replication process. In parallel, co-assembly on all five samples is performed](./images/individual-assembly.png "Individual assembly followed by de-replication vs co-assembly"){:width="80%"}
+
+If we have different samples, then we do an individual assembly for each sample. In the figure above we see that after individual assembly we have results for every individual assembly represented with pie charts. Different colours on these charts show different strains (organisms). Every chart has a different percentage of every strain which means that the assemblies contain different strains in different proportions in each sample.
+
+Afterwards, we do the process of de-replication. We try to combine all the assemblies and try to identify which genomes are the most proper.
+
+Individual assembly is a good practice as well as co-assembly. They both have pros and cons and that are just different techniques.
+
+Co-assembly is a more common practice. But in case of co-assembly the genome might be more fragmented afterwards (like it is shown in the figure)  and sometimes it can be less proper. However, it should be decided in every single case which approach to use (co- or individual). More comprehensive information about de-replication you can learn from paper {%cite evans2020%} to get more intuition about how de-replication works.
+
+> <hands-on-title>General list of actions for de-replication</hands-on-title>
+> 1. Create new history
+> 2. Assemble each sample separately using your favorite assembler
+> 3. Perform a co-assembly to catch low-abundance microbes
+> 4. Bin each assembly separately using your favorite binner
+> 5. Bin co-assembly using your favorite binner
+> 6. Pull the bins from all assemblies together
+> 7. rRun **dRep** on them
+> 8. Perform downstream analysis on the de-replicated genome list
+>
+{: .hands_on}
+
+We will perform steps from 1 to 3 in this tutorial a bit later while steps 4 - 8 will be considered in the following tutorial - Binning tutorial.-->
+
+# Conclusions
+
+In summary, this tutorial shows a step-by-step on how to bin metagenomic contigs using MetaBAT2.
+
+It is critical to select the appropriate binning tool for a specific metagenomics study, as different binning methods may have different strengths and limitations depending on the type of metagenomic data being analyzed. By comparing the outcomes of several binning techniques, researchers can increase the precision and accuracy of genome binning.
+
+There are various binning methods available for metagenomic data, including reference-based, clustering-based, and hybrid approaches. Each method has its advantages and disadvantages, and the selection of the appropriate method depends on the research question and the characteristics of the data.
```

Comment:
You mentioned only 2 in the introduction

Reply:
Binning methods have been updated in order to be more specific